discovery-infra/test_infra/controllers/node_controllers/node.py

```diff
@@ -145,3 +145,12 @@ def attach_test_disk(self, disk_size, bootable=False):
 
     def detach_all_test_disks(self):
         self.node_controller.detach_all_test_disks(self.name)
+
+    def attach_interface(self, node_name, network_xml, target_interface="vnet3"):
+        return self.node_controller.attach_interface(node_name, network_xml, target_interface)
+
+    def destroy_network(self, network):
```

Comment:
I don't think destroy network should be at `node` level
How about moving into `nodes`?

Reply:
Well i agree that network is more nodes related and i dont mind to change it. But isnt it more intuitive to `node.attach_interface`? i mean, what reason do we have to create a network w/o attaching it to a specific node interface? 
this also saves the need to pass and save both nodes and node (on test level, and on base).

Reply:
`node.attach_interface` can get network name of an already created interface that you can attach another interface on the same network

Reply:
Fixed the code by allowing attach_interface fixture to support network and interface creation OR interface creation by given existing network name. 

I think network need to be under node, its more intuitive for me.
Also if changing to nodes, than attach_interface method should be located under nodes, which is also not accurate. 
LMK if you disagree